app/src/main/java/com/example/lwilcox/lab2/FeedReaderDbHelper.java

```diff
@@ -0,0 +1,87 @@
+package com.example.lwilcox.lab2;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class FeedReaderDbHelper extends SQLiteOpenHelper{
+    public static final int DATABASE_VERSION = 1;
+    public static final String DATABASE_NAME = "FeedReader.db";
+
+    public FeedReaderDbHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+    public void onCreate(SQLiteDatabase db) {
+        db.execSQL(FeedEntry.SQL_CREATE_ENTRIES);
+    }
+
+    public void addItem(String item) {
+        SQLiteDatabase db = getWritableDatabase();
```

Comment:
You could improve performance if you declare this variable at the top of the class and then intialize it in the constructor. That way you only need to call getWrtiableDatabase once. Instead, you have to call it every time you wish to addItem